api/models/user.py

```diff
@@ -1,6 +1,6 @@
 from api.mixins import TimestampMixin
 from api import db
-from flask_sqlalchemy import BaseQuery
+from flask_sqlalchemy.query import Query as BaseQuery

```

Comment:
The `BaseQuery` class has been renamed as `query` now and is situated within the query module.